quantum/quantum_keycodes.h

```diff
@@ -475,6 +475,11 @@ enum quantum_keycodes {
     HPT_DWLI,
     HPT_DWLD,
 
+    MAGIC_SWAP_LCTL_LGUI,
```

Comment:
Ah, yeah, here is the issue.  It looks like your repo may be out of date. 

This should have: 

```suggestion
    // Left control, open paren
    KC_LCPO,

    // Right control, close paren
    KC_RCPC,

    // Left control, open paren
    KC_LAPO,

    // Right control, close paren
    KC_RAPC,
    
    MAGIC_SWAP_LCTL_LGUI,
```